app/components/Stock/tabs/StockNews.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+import { Text, View } from 'native-base';
+import theme from '../../../theme';
+import { formatTimeFromNow } from '../../../utils/helpers';
+import * as Actions from '../store/actions';
+
+export function StockNews(props) {
+  const dispatch = useDispatch();
+  const symbol = useSelector(({ stock }) => stock.data.symbol);
+  const data = useSelector(({ stock }) => stock.news.data);
+
+  useEffect(() => {
+    dispatch(Actions.getNewsData(symbol));
+  }, [dispatch, symbol]);
+
+  return data.map(item => (
```

Comment:
you can destruct item here as well

Reply:
resolved 32f2911b404e4cc45ba8178ff0731e39a308a31d